### PR TITLE
feat(icon): add invoice icon

### PR DESCRIPTION
**Pull request by smehdii**

### CODE REVIEW

File: docs/stories/icon.stories.js [hunk 1]
```diff
@@ -173,13 +175,20 @@ storiesOf('API|Icon', module)
...
-      {[EuroIcon, ForbiddenIcon, FlatrateSt1Icon, FlatrateSt2Icon, FlatrateSt3Icon, FlatrateTsIcon, FlatrateTdIcon].map(
-        (Icon, index) => (
-          <Showcase variant="squared" key={index}>
-            <Icon circle circleColor={palette.mediumBeetrootPurple} color="white" />
-          </Showcase>
-        )
-      )}
+      {[
+        EuroIcon,
+        InvoiceIcon,
```
Comment (chabou) [inline docs/stories/icon.stories.js]: I don't think it is relevant

File: packages/andive/src/components/icons/invoice.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import Invoice from '../../../img/icons/invoice.svg'
+import Icon from '../icon'
+
+const InvoiceIcon = props => {
+  return (
+    <Icon {...props}>
+      <Invoice />
+    </Icon>
+  )
+}
```
Comment (chabou) [inline packages/andive/src/components/icons/invoice.js:20]: All other squared icon can have a color prop
https://github.com/amblerhq/andive/blob/cb6ba73a84cdb42f675945bc50cd19ea18a7b2c0/packages/andive/src/components/icons/mf.js#L8-L12

**Review by chabou — CHANGES_REQUESTED** [review]
InvoiceIcon should be quite the same than mfIcon

File: docs/stories/icon.stories.js (hunk 1 above)
Comment (smehdii) [reply docs/stories/icon.stories.js]: EuroIcon used in invoicesCard , and InvoiceIcon used in the menuDrawer as discussed earlier.

**Review by chabou — APPROVED** [review]

**Pull request merged**